README.md

```diff
@@ -87,7 +87,7 @@ Other Style Guides
 
     bar[0] = 9;
 
-    console.log(foo[0], bar[0]); // => 9, 9
+    console.log(foo[0], bar[0]); // => 1, 9
```

Comment:
In fact it will be `9, 9`, which is the sole purpose of this guide section. Try it in the console and see!